README.md

```diff
@@ -197,6 +201,21 @@ There are a number of inconsistencies in how browsers handle form events, valida
 
 ----
 
+<a name="upgrade-to-v2"></a>
+
+## Upgrading from v1.x.x
+
+The main change in `v2` is that classes provided by `o-forms` now conform more strictly to the [BEM naming convention][bem]. All form field classes now follow the element convention, so `o-forms-text` is now `o-forms__text`.
+
+There is also now a main block class of `o-forms` which replaces the previous `o-forms-group` class. Full class changes are below:
+
+- `o-forms-group` becomes `o-forms`
```

Comment:
Yes it's only the classes that are changing. There's some new Sass mixins but none of the originals have changed name or function